### DISCUSSION TITLE
A question about the continuation of a failed export.

### DISCUSSION THREAD

**Discussion by BG-Cypo** [Tyrrrz/DiscordChatExporter#724] [Q&A] [answered]
If exporting a channel failed, but the medias and the .HTML remains, if I export it again, would it continue from the last export or would it start again?

**Comment by 96-LB** [comment, marked as answer]
There is no way to continue a partial export. However, you can use the "Reuse downloaded media" option (`--reuse-media` flag on CLI) to prevent downloading media files a second time. Since this often requires the bulk of the time and data, you can substantially speed up the part of the export which was already completed.

**Reply by BG-Cypo** [reply]
I use the GUI version so I assume that it is the same as enabling it in the Settings. Thank you!

**Reply by 96-LB** [reply]
Yup! It's the one called "Reuse downloaded media" on the GUI.
